pennylane_lightning/lightning_gpu/_state_vector.py

```diff
@@ -289,7 +311,14 @@ def _apply_lightning(
             method = getattr(state, name, None)
             wires = list(operation.wires)
 
-            if method is not None:  # apply specialized gate
+            if isinstance(operation, Conditional):
+                if operation.meas_val.concretize(mid_measurements):
+                    self._apply_lightning([operation.base])
```

Comment:
Do we need `operation.base` as a list?

Reply:
Why not? `_apply_lightning` accepts list[~operations]